### PR TITLE
Aboutlicencefragment #189

### PR DESCRIPTION
**Pull request by wischweh**
icon for Show Licence Button still missing
Screen and text for licences are still missing

### CODE REVIEW

File: qabelbox/src/main/java/de/qabel/qabelbox/adapter/JSONLicencesAdapter.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                    if (content.length() > 0 && componentsJSON.length() > 0) {
+                        content = content.substring(0, content.lastIndexOf("<br/>")); // Strip last linebreak
+                        content.trim();
+                    }
+                }
+                SpannableString formattedText = new SpannableString(Html.fromHtml(content));
+                this.content.setText(formattedText, TextView.BufferType.SPANNABLE);
+                this.showLicenceBtn.setOnClickListener(this);
+            } catch (JSONException e) {
+                e.printStackTrace();
```
Comment (audax) [inline qabelbox/src/main/java/de/qabel/qabelbox/adapter/JSONLicencesAdapter.java]: Do this with Log.e(TAG, e)

File: qabelbox/src/main/java/de/qabel/qabelbox/adapter/JSONLicencesAdapter.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            }
+        }
+
+        @Override
+        public void onClick(View v) {
+            // TODO: Implement Licence Screen
+            Toast.makeText(ctx, "Hier könnte Ihre Lizenz stehen", Toast.LENGTH_SHORT).show();
```
Comment (audax) [inline qabelbox/src/main/java/de/qabel/qabelbox/adapter/JSONLicencesAdapter.java]: i18n missing

File: qabelbox/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -216,4 +216,13 @@
     <string name="upload_complete_notification_title">Upload complete</string>
     <string name="upload_in_progress_notification_content">Upload in progress: %s</string>
     <string name="message_file_cant_upload">File can\'t upload</string>
+
+    <string name="licence.header.intro">Qabel is Published under the QAPL:</string>
+    <string name="about.header.qabel.company.qtxt">QABEL</string>
+    <string name="about.header.qabel.show.btn">Show QAPL</string>
+    <string name="about.header.qabel.thanks.header.txt"> bedankt sich bei:</string>
```
Comment (audax) [inline qabelbox/src/main/res/values/strings.xml]: i18n

File: qabelbox/src/main/java/de/qabel/qabelbox/adapter/JSONLicencesAdapter.java (hunk 1 above)
Comment (wischweh) [inline qabelbox/src/main/java/de/qabel/qabelbox/adapter/JSONLicencesAdapter.java]: done

File: qabelbox/src/main/java/de/qabel/qabelbox/adapter/JSONLicencesAdapter.java (hunk 2 above)
Comment (wischweh) [inline qabelbox/src/main/java/de/qabel/qabelbox/adapter/JSONLicencesAdapter.java]: i18n does not make sense here, as it is just as stub for the future implementation of the licence screen

**Comment by wischweh** [comment]
re-merged and integrated an "about" button into the drawer

**Pull request merged**